tests/basic/longevity_test.go

```diff
@@ -102,23 +103,35 @@ var _ = Describe("{Longevity}", func() {
 
 	BeforeEach(func() {
 		if !populateDone {
+			StartTorpedoTest("PX-Longevity", "Validate PX longevity workflow", nil)
+
 			populateIntervals()
 			populateDisruptiveTriggers()
 			populateDone = true
 		}
 	})
 
 	It("has to schedule app and introduce test triggers", func() {
-		Step(fmt.Sprintf("Start watch on K8S configMap [%s/%s]",
-			configMapNS, testTriggersConfigMap), func() {
+		dash.Info("has to schedule app and introduce test triggers")
```

Comment:
can you make this message a little more meaning full

Reply:
fixed